### PR TITLE
Show image under status bar in plant details screen 

### LINKED ISSUE #869: Show image under status bar in plant details screen
As an improvement for Layout, we can make the status bar transparent on the Details screen as the status bar can be part of the layout, 

### What is currently showing is  👇🏻 : 

Screenshot 2023-06-28 at 4 59 39 PM

### What I am suggesting is 👇🏻 : 

Screenshot 2023-06-28 at 5 01 41 PM

## 

Part of [GTC](https://www.facebook.com/groups/gazatechcommunity/) open source initiative

### PR DESCRIPTION
**Pull request by qamarelsafadi**
## What?

Fixes #869 

## Why?

This pull request addresses the issue of not showing the image under the status bar as a result that the status bar was not transparent and the Image inside the PlantDetails Screen was having SystemBars padding which added padding top and bottom of the whole screen. 

## How?

To achieve this result 👇🏻 : 

Screenshot 2023-06-30 at 2 04 49 PM
 
## 
I had to add a new dependency to have control of the System UI Controller which is the Status bar and Navigation bar, 
after adding the new dependency I had to make the Status bar transparent and remove the systemBarsPadding Modifier from PlantDetails to let the image showing under the Status bar

## 

This PR is part of [GTC](https://www.facebook.com/groups/gazatechcommunity/) open source initiative

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -27,6 +27,7 @@
 # The setting is particularly useful for tweaking memory settings.
 android.useAndroidX=true
 org.gradle.jvmargs=-Xmx1536m
+unsplash_access_key=W4WyLAjz2Lr5O_EV0UtujNvzjTtlUtWlYYGirbrKcWU
```
Comment (arriolac) [inline gradle.properties:30]: Remove

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -118,6 +119,7 @@ kotlinx-coroutines-core = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-c
 kotlinx-coroutines-guava = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-guava", version.ref = "coroutines" }
 kotlinx-coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "coroutines" }
 material = { module = "com.google.android.material:material", version.ref = "material" }
+accompanist = { module = "com.google.accompanist:accompanist-systemuicontroller", version.ref = "systemuicontroller" }
```
Comment (arriolac) [inline gradle/libs.versions.toml:122]: This name should be more specific like `accompanist-systemuicontroller`

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt [hunk 3]
```diff
@@ -200,7 +200,7 @@ fun PlantDetails(
     Box(
         modifier
             .fillMaxSize()
-            .systemBarsPadding()
+//            .systemBarsPadding()
```
Comment (arriolac) [inline app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:203]: Delete instead

File: app/src/main/java/com/google/samples/apps/sunflower/ui/Theme.kt [hunk 4]
```diff
@@ -69,11 +72,25 @@ fun SunflowerTheme(
...
+        DisposableEffect(systemUiController, useDarkIcons) {
+            // Update all of the system bar colors to be transparent, and use
+            // dark icons if we're in light theme
+            systemUiController.setSystemBarsColor(
+                color = Color.Transparent,
+                darkIcons = useDarkIcons
+            )
+            onDispose {}
+        }
+        /*  SideEffect {
```
Comment (arriolac) [inline app/src/main/java/com/google/samples/apps/sunflower/ui/Theme.kt:88]: Delete this instead

**Review by arriolac — COMMENTED** [review]
Thanks for the contribution! Left some comments

File: app/src/main/java/com/google/samples/apps/sunflower/ui/Theme.kt (hunk 4 above)
Comment (qamarelsafadi) [reply app/src/main/java/com/google/samples/apps/sunflower/ui/Theme.kt]: Done

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt (hunk 3 above)
Comment (qamarelsafadi) [reply app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt]: Done

File: gradle/libs.versions.toml (hunk 2 above)
Comment (qamarelsafadi) [reply gradle/libs.versions.toml]: Done

File: gradle.properties (hunk 1 above)
Comment (qamarelsafadi) [reply gradle.properties]: Done

**Review by arriolac — APPROVED** [review]
LGTM

**Pull request merged**